internal/uvm/scsi.go

```diff
@@ -194,20 +232,58 @@ func (uvm *UtilityVM) AddSCSI(ctx context.Context, hostPath string, uvmPath stri
 // `uvmPath` is optional if a guest mount is not requested.
 //
 // `readOnly` set to `true` if the physical disk should be attached read only.
-func (uvm *UtilityVM) AddSCSIPhysicalDisk(ctx context.Context, hostPath, uvmPath string, readOnly bool) (*SCSIMount, error) {
-	return uvm.addSCSIActual(ctx, hostPath, uvmPath, "PassThru", false, readOnly)
+func (uvm *UtilityVM) AddSCSIPhysicalDisk(ctx context.Context, hostPath, uvmPath string, readOnly bool) (sm *SCSIMount, err error) {
+	sm, existed, err := uvm.allocateSCSIDevice(ctx, hostPath, uvmPath, VMAccessTypeIndividual)
```

Comment:
I think there's enough code here to justify keeping the helper function, so that `AddSCSI` and `AddSCSIPhysicalDisk` both call into it.

Reply:
which helper function are you referring to?

Reply:
Previously `AddSCSI` and `AddSCSIPhysicalDisk` did nothing except call into `addSCSIActual`. I think we should reduce the current duplication between these two functions.

Reply:
I see. I moved out the allocateSCSIDevice from the addSCSIActual function to allow the AddSCSILayers function access to the controller and lun for constructing the uvmPath. I understand there's code dup here, but if we plan to move the more in depth AddSCSILayer refactoring/removal to another PR, is this fine here?

Reply:
Yes, we can do this later.